addons/test_discuss_full/tests/test_performance.py

```diff
@@ -147,7 +147,7 @@ def _get_init_messaging_result(self):
                                 'active': True,
                                 'email': 'e.e@example.com',
                                 'id': self.users[0].partner_id.id,
-                                'im_status': 'online',
+                                'im_status': self.users[0].partner_id.im_status,
```

Comment:
Hello, this is not critical but please keep an hard-coded value in this test, the `im_status` of the user is supposed to be know according to the test setup.

You can add `hr_homeworking` as dependency of this module if necessary to make it reliable depending on setup.